planner/core/find_best_task.go

```diff
@@ -30,7 +30,7 @@ const (
 	netWorkFactor      = 1.5
 	netWorkStartFactor = 20.0
 	scanFactor         = 2.0
-	descScanFactor     = 2 * scanFactor
+	descScanFactor     = 2 * scanFactor //2 is the descscan requires scan twice
```

Comment:
> requires scan twice

No, it's not.

Reply:
@winoros @zhangjinpeng1987 PTAL

Reply:
These factors here are used to estimate the cost of a physical plan. Here 2 means descend scan will be slower than ascend scan, descend scan will takes 2 times of the time compare to ascend scan for scan the same number of rows.